lib/service/cfg.go

```diff
@@ -877,6 +878,10 @@ type LDAPConfig struct {
 	Username string
 	// Password for LDAP authentication.
 	Password string
+	// SkipVerifyCA decides whether whether we skip verifying the CA when making the LDAPS connection.
```

Comment:
Technically we're not verifying the CA, we're verifying the LDAP server certificate _with the CA_. I would find a more accurate way to phrase this.